retworkx/__init__.py

```diff
@@ -648,12 +648,12 @@ def _graph_all_pairs_dijkstra_shortest_path(graph, edge_cost_fn):
 
 @functools.singledispatch
 def all_pairs_dijkstra_path_lengths(graph, edge_cost_fn):
-    """Find the shortest path from a node
+    """Calculate the shortest path lengths from all nodes in a graph
```

Comment:
```suggestion
    """For each node in the graph, calculates the lengths of the shortest paths to all others.
```
This wording assumes the key of the result map is the source and the key of the value map is the target.

Reply:
Yeah, that's how the function works this is definitely more clear than what I had here